com.unity.perception/Runtime/GroundTruth/GroundTruthCrossPipelinePass.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace UnityEngine.Perception.GroundTruth
 {
-    abstract class GroundTruthCrossPipelinePass : IGroundTruthGenerator
+    public abstract class GroundTruthCrossPipelinePass : IGroundTruthGenerator
```

Comment:
This needs to be documented if it is to be made public. Probably easier to just make it internal again though.